pkg/cmd/run.go

```diff
@@ -67,8 +63,7 @@ func NewRunMicroshiftCommand() *cobra.Command {
 }
 
 func RunMicroshift(cfg *config.MicroshiftConfig, flags *pflag.FlagSet) error {
-
-	if err := cfg.ReadAndValidate(flags); err != nil {
+	if err := cfg.ReadAndValidate("", flags); err != nil {
```

Comment:
I think instead of having ReadAndValidatate() be responsible for setting a default, this higher level function should do it. We can fix that in a follow-up PR, though.